test/services/school/index.test.js

```diff
@@ -395,6 +386,22 @@ describe('school service', () => {
 			}
 			expect(result2.officialSchoolNumber).to.be.equal(schoolNumber2);
 		});
+		it('should fail to update county if state does not have the provided county ', async () => {
```

Comment:
I miss one test for the use case of this new countrys ..maybe in schools. It make it more cleaner for me, how it should use and make the data structure sense in this way.

Reply:
done